apps/starwars/web/graphql/star-wars/types.ex

```diff
@@ -10,7 +10,7 @@ defmodule Webapp.GraphQL.StarWars.Types do
   use Absinthe.Relay.Schema.Notation
 
   alias Absinthe.Relay.Connection
-  alias Webapp.GraphQL.StarWarsDB
+  alias Starwars.GraphQL.StarWarsDB
```

Comment:
Update the alias based on a few comments above. Also notice it should be StarWars and not Starwars